velox/flag_definitions/flags.cpp

```diff
@@ -131,3 +131,6 @@ DEFINE_bool(
     velox_ssd_verify_write,
     false,
     "Read back data after writing to SSD");
+
+// Used in /connectors/tpch
+DEFINE_int32(velox_tpch_text_pool_size_mb, 300, "TPC-H pool size in MB.");
```

Comment:
nit: `"TPC-H DBGen text pool size in MB"`  Other flag comments don't end in a fullstop.